t/etc/redfishpower-setplugs.dev

```diff
@@ -0,0 +1,48 @@
+# Variant of redfishpower-cray-r272z30.dev that covers use of setplugs
+# configuration
+specification "redfishpower-setplugs" {
+	timeout 	60
```

Comment:
Would it be helpful to develop a sharness test that runs `redfishpower` standalone rather than through powerman?  This test seems awfully complicated just to check that setplugs works, and presumably we'll eventually have a dev script that uses all the new features in test (like the one I proposed).

Reply:
yeah, the main reason I ran through powerman is b/c there isn't a facility to run `redfishpower` someone independently of powerman at the moment. 

For the time being, I'll splice out the "redfishpower" specific tests into a new file, rather than keep it within the "redfish" set of tests.  Since they are clearly of a different beast than the device files in /etc.

Reply:
Probably I'm oversimplifying again :-)

I was vaguely thinking you could write a sharness test that drove `redfishpower` with a here-is document in a sharness script but maybe that's not practical.